Calling/ClientApp/src/core/actions/calls.ts

```diff
@@ -1,11 +1,13 @@
 import { CallEndReason, Call, RemoteParticipant, CallAgent } from '@azure/communication-calling';
+import { SelectionState } from '../RemoteStreamSelector';
 
 const SET_CALL_AGENT = 'SET_CALL_AGENT';
 const SET_GROUP = 'SET_GROUP';
 const CALL_ADDED = 'CALL_ADDED';
 const CALL_REMOVED = 'CALL_REMOVED';
 const SET_CALL_STATE = 'SET_CALL_STATE';
 const SET_PARTICIPANTS = 'SET_PARTICIPANTS';
+const SET_SELECTED_PARTICIPANTS = 'SET_SELECTED_PARTICIPANTS';
```

Comment:
selected participants makes sense to us in this problem but I think we need to rethink what this name should be "selected participants" for anyone else reading what is going on. Happy to brainstorm here.

Reply:
Agree, maybe SET_SELECTED_REMOTE_STREAMS or SET_REMOTE_STREAMS_TO_RENDER?

Reply:
changed to SET_DOMINANT_PARTICIPANTS.